File: grupo_5_proyecto_2026_fronted/src/types/MenuItem.ts
export type MenuItem = {
  id: number;
  name: string;
  description?: string | null;
  price?: number | null;
  photo?: string | null;
  isactive?: boolean | null;
};

export type MenuItemFormData = {
  name: string;
  description: string;
  price: string;
  isactive: boolean;
  photoFile: File | null;
};

File: grupo_5_proyecto_2026_backend/FourLoveRestaurant/Models/Menuitem.cs
using System;
using System.Collections.Generic;

namespace FourLoveRestaurant.Models;

public partial class Menuitem
{
    public int Id { get; set; }

    public string Name { get; set; } = null!;

    public string? Description { get; set; }

    public decimal? Price { get; set; }

    public string? Photo { get; set; }

    public bool? Isactive { get; set; }
}
